neuvector-monitor-5.3.yaml

```diff
@@ -27,16 +29,10 @@ pipeline:
 
   - runs: |
       make -C monitor
-      mkdir -p ${{targets.contextdir}}/usr/local/bin
-      install -Dm755 monitor/monitor ${{targets.contextdir}}/usr/bin/neuvector-monitor
-
-subpackages:
-  - name: ${{package.name}}-compat
-    description: NeuVector Monitor binary symlink
-    pipeline:
-      - runs: |
-          mkdir -p ${{targets.contextdir}}/usr/local/bin
-          ln -s /usr/bin/neuvector-monitor ${{targets.contextdir}}/usr/local/bin/monitor
+      mkdir -p ${{targets.destdir}}/usr/local/bin
```

Comment:
you are using `destdir` here, that could be `contextdir` to avoid misconfigurations for targets.